grasa_event_locator/static/css/custom.css

```diff
@@ -425,6 +425,22 @@ span.subhead::before, span.subhead::after {
 	position:relative;
 }
 
+/*Styles for HomePage title*/
+.jumbotron {
+	background-color: #EFFBEF;
+	position: relative;
+
+}
+
+.jumbotron p {
+	text-align:center;
+}
+
+.alert {
```

Comment:
Do not add style directly to the alert class, this will affect all alerts across the app. You can give the alert you want to change a specific class and then add styles to that.